osu.Game/Beatmaps/Drawables/Cards/BeatmapCard.cs

```diff
@@ -366,8 +368,6 @@ protected override bool OnHover(HoverEvent e)
 
         protected override void OnHoverLost(HoverLostEvent e)
         {
-            content.ScheduleHide();
```

Comment:
This is the single cause of the failure. `OnHoverLost` is fired when one of the child handles it.

Everything else is just cleaning things up.